docker-compose.yaml

```diff
@@ -19,11 +19,28 @@ services:
       - 5671:5671
       - 5672:5672
 
-  couchbase-server-sandbox:
-    image: couchbase/server-sandbox:7.1.1
+  couchbase:
```

Comment:
I think we need to test this new couchbase config in sonarqube github action as well.

Reply:
Yes. I think it will work fine as we start the `couchbase` container using the docker-compose file in GitHub action.